test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -304,6 +304,31 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 				Expect(processesFound).To(BeEmpty(), "The node %s has the following processes on the root cgroup: %v", node.Name, processesFound)
 			}
 		})
+
+		It("Check for agetty process system.slice cgroup", func() {
+			for _, node := range workerRTNodes {
+				agettyPidCommand := "pgrep agetty"
+				agettyPidOutput, err := nodes.ExecCommandOnNode([]string{"/bin/bash", "-c", agettyPidCommand}, &node)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(agettyPidOutput).ToNot(BeEmpty())
+				agettyPids := strings.Fields(agettyPidOutput)
+				Expect(len(agettyPids)).To(BeNumerically(">", 0), "No agetty PIDs found on node %s", node.Name)
```

Comment:
also: ```Expect(agettyPids).ToNot(BeEmpty())```

Reply:
Hmm.. for making the test meaninful yes. But is missing agetty really a bug?